AutomatedTesting/Gem/PythonTests/Physics/CMakeLists.txt

```diff
@@ -12,18 +12,6 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
         TEST_SUITE main
         TEST_SERIAL
         PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Main.py
-        RUNTIME_DEPENDENCIES
-            Legacy::Editor
-            AZ::AssetProcessor
-            AutomatedTesting.Assets
-        COMPONENT
-            Physics
-    )
-    ly_add_pytest(
-        NAME AutomatedTesting::PhysicsTests_Main_Optimized
```

Comment:
What happens to the main optimized test suite with this change? Is it basically replacing the main test suite?

Reply:
Yes. That's what Alex's PR did, get rid of main and use main optimized instead (moved main optimized -> main).